### PR TITLE
Feature/game

### PR DESCRIPTION
**Pull request by MylesNottingham**
This will establish the Game class.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -9,8 +9,23 @@ def self.from_csv(files)
...
+    highest_game.away_goals + highest_game.home_goals
+  end
+
+  def lowest_total_score
+    lowest_game = @games.min_by do |game|
+      game.away_goals + game.home_goals
     end
+    lowest_game.away_goals + lowest_game.home_goals
   end
+  ### LEAGUE STATS ###
```
Comment (javenb022) [inline lib/stat_tracker.rb:29]: Gonna add some more features here and under team stats

**Review by javenb022 — APPROVED** [review]
Looks good team

File: lib/game.rb [hunk 2]
```diff
@@ -16,8 +16,8 @@ def initialize(row)
     @date_time = row[:date_time]
     @away_team_id = row[:away_team_id]
     @home_team_id = row[:home_team_id]
-    @away_goals = row[:away_goals]
-    @home_goals = row[:home_goals]
+    @away_goals = row[:away_goals].to_i
```
Comment (garrettgregor) [inline lib/game.rb:19]: Good call here!!

Myles == genius
=> true

**Review by garrettgregor — APPROVED** [review]
Looks great Myles!

**Pull request merged**